app/controllers/katello/api/v2/activation_keys_controller.rb

```diff
@@ -293,6 +294,10 @@ def verify_presence_of_organization_or_environment
       fail HttpErrors::BadRequest, _("Either organization ID or environment ID needs to be specified")
     end
 
+    def find_content_view
+      @content_view = ContentView.find_by(:id => params[:content_view_id])
```

Comment:
This should probably throw an error like [1] if no content_view found.

[1] https://github.com/Katello/katello/pull/5995/files#diff-65d2a2cbc038a622df645394a941f518R286